packages/cra-template/.npmrc

```diff
@@ -0,0 +1,2 @@
+registry=https://registry.npmjs.org/
```

Comment:
shouldn't this be the npm default ?

Reply:
Hmm, this file was inserted when we ran `yarn create-package` from the root, so not sure about this. Maybe the default behaviour should be changed?

Reply:
It seems this config file is inserted in every package within `packages/` source dir.

Reply:
FYI, @Hotell just found it in [`scripts/create-package/plop-templates`](https://github.com/microsoft/fluentui/tree/master/scripts/create-package/plop-templates).

Reply:
Perhaps we need some follow up in the future to change this, it's definitely not needed anywhere.

Not a showstopper for this PR tho, feel free to resolve

Reply:
Yeah the scaffolding tool should be revisited

Reply:
@joschemd has been working on the create-package templates, follow up with him on the suggested change (perhaps should file an issue to track it too)

Reply:
@kenotron Do you remember why we needed this? (or if we still do?)

Reply:
Probably best to leave the .npmrc file for now, and then if we determine it's unnecessary we can remove them all together.

Reply:
On closer inspection, I think the npmrc is to be able to publish through CI, so that beachball can set the NPM_TOKEN hack to auth with the registry:

https://blog.npmjs.org/post/118393368555/deploying-with-npm-private-modules

Reply:
we publish private packages ? that's seems odd. curios what is the use case ?

Reply:
Our standard publishing job for Fluent UI doesn't publish private packages. @ling1726 I'm a bit confused what you meant since I don't see NPM_TOKEN anywhere in our code or beachball's code.

Reply:
And @dzearing was pretty sure it's okay to get rid of the per-package .npmrc. So if someone would like to make another PR doing that feel free.